packages/design-system/hooks/index.ts

```diff
@@ -1,14 +1,13 @@
 /* eslint-disable react-hooks/exhaustive-deps */
-import { useRef, useEffect, useState, useMemo, useLayoutEffect } from "react";
+import { useRef, useEffect, useState, useMemo } from "react";
 import { LayoutChangeEvent, Platform } from "react-native";
 
 import { useSharedValue } from "react-native-reanimated";
 
 import { useColorScheme } from "@showtime-xyz/universal.color-scheme";
 import { getScrollParent } from "@showtime-xyz/universal.utils";
 
-const useIsomorphicLayoutEffect =
-  typeof window !== "undefined" ? useLayoutEffect : useEffect;
+import { useIsomorphicLayoutEffect } from "app/hooks/use-isomorphic-layout-effect";
```

Comment:
I think our design system folder components shouldn't rely on any `app` folder files. Otherwise, if we want to release them someday, it will be troublesome. wdty?

Reply:
oh yes, totally missed it's importing in design folder